applications/nrf5340_audio/unicast_client/main.c

```diff
@@ -354,6 +369,20 @@ static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
 	case BT_MGMT_CONNECTED:
 		/* NOTE: The string below is used by the Nordic CI system */
 		LOG_INF("Connection event. Num connections: %u", num_conn);
+
+		struct bt_le_conn_param param;
+
+		/* Set a shorter ACL interval to speed up discovery and configuration */
+		param.interval_min = CONFIG_BLE_ACL_CONN_INTERVAL_FAST;
+		param.interval_max = CONFIG_BLE_ACL_CONN_INTERVAL_FAST;
+		param.latency = CONFIG_BLE_ACL_SLAVE_LATENCY;
+		param.timeout = CONFIG_BLE_ACL_SUP_TIMEOUT;
+
+		ret = bt_conn_le_param_update(msg->conn, &param);
+		if (ret) {
+			LOG_WRN("Failed to update conn parameters: %d", ret);
+		}
+
```

Comment:
As a Bluetooth Central we can set the initial ACL interval when connecting to avoid running a connection update after connection setup. Should that be done instead? Or are we here trying to improve the situation where the unicast client is a Bluetooth peripheral?

Reply:
As discussed, I don't want non-ISO related ACls to fall in to the default of having 10ms, so default should be larger as it is less intrusive on air and then we increase the interval when we know that a lot of data will go over air.

Reply:
After more discussions with @koffes I ended up setting the default to 10, and then setting it to 90 later. We need to do some more thinking related to supporting both 7.5 and 10, as well as non-ISO related ACL devices.